lib/samson/hooks.rb

```diff
@@ -78,7 +78,9 @@ def callback(name, &block)
       end
 
       def view(name, partial)
-        hooks(name) << partial
+        hooks(name) do |view, *args|
```

Comment:
before we stored the name of the partial and did the rendering logic in render_views,
this would change it to store the rendering logic in the hook (to be consistent with other hooks that also only store lambdas)

consistency is nice, but simple inspectable data is too ... so choose wisely :)